app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.first
+  end
```

Comment:
- [ ] Like the last code reviewer said kindly delete the current user method. Because it will come from devise now. It is part of the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Remove%20current_user%20method%20in%20ApplicationController%2C%20devise%20will%20provide%20us%20with%20one.).

Reply:
This issue is well resolved! ✔️